src/docker/workflows/confluent-local.ts

```diff
@@ -101,7 +102,7 @@ export class ConfluentLocalWorkflow extends LocalResourceWorkflow {
     }
     count = parseInt(numContainersString, 10);
     const plural = count > 1 ? "s" : "";
-    this.sendTelemetryEvent("Input Box Filled", {
+    this.sendTelemetryEvent(UserEvent.InputBoxFilled, {
```

Comment:
Not related to this PR but why do we collect this event? Seems random.

Reply:
Originally it was just to gauge when someone filled out the "how many brokers/containers" versus exiting early, but I'm good with removing it entirely if we want

Reply:
oh that does seem useful; makes sense